core/src/main/kotlin/com/linecorp/kotlinjdsl/querydsl/QueryDslImpl.kt

```diff
@@ -129,8 +130,20 @@ open class QueryDslImpl<T>(
         lazyJoins().add(FetchJoinSpec(left = left, right = right, path = relation.path, joinType = joinType))
     }
 
-    override fun where(predicate: PredicateSpec) {
-        lazyWheres().add(predicate)
+    override fun where(predicate: PredicateSpec?) {
+        predicate?.let { lazyWheres().add(it) }
```

Comment:
Since it is not a conversion concept, how about changing it to run rather than let?

-- korean
변환하는 개념이 아니라서 let 보다는 run 으로 바꾸시는게 어떤가요?

Reply:
[refactor: using 'run' instead of 'let'](https://github.com/line/kotlin-jdsl/pull/75/commits/2ade308b47376be688cb6ae2cb2ca88f15fb2d71) 커밋에 적용하였습니다
테스트 통과 되는 것도 확인하였습니다
감사합니다